tests/system/fim/common_tasks/verify_alerts_json.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python3
+
+# Copyright (C) 2015-2020, Wazuh Inc.
+# All rights reserved.
+#
+# This program is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation.
+
+import sys
+import json
+
+if sys.version_info.major < 3:
+    print('ERROR: Python 2 is not supported.')
+    sys.exit(1)
+
+
+def paths_acquisition(filenames_list_path):
+    """
+        Turn path lists file into a set.
+        :param str filenames_list_path: path to filenames list
+        :return: Returns a set containing the generated files path
+    """
+    filenames_list = []
+    with open(filenames_list_path) as lines:
+        filenames_list = lines.readlines()
+    return set([i[:-1] for i in filenames_list])
+
+
+def alerts_prune(path, target_event):
+    """
+        Prunes desired syscheck events from the alert.json file.
+        Extracts all events path to a set.
+        :param str path: path to alerts.json file
+        :param str target_event: target event kind (deleted|added|modified)
+        :return: Returns a set containing the alerts files path
+    """
+    alerts_list = []
+    with open(path) as json_file:
+        for line in json_file:
+            data = json.loads(line)
+            if data.get('syscheck') and data['syscheck']['event'] == target_event:
+                alerts_list.append(data)
+    return set([alerts['syscheck']['path'] for alerts in alerts_list])
+
+
+def main():
+
+    import argparse
+    parser = argparse.ArgumentParser(description='Compares paths list and alerts.json paths')
+
+    parser.add_argument("-f", "--files", type=str, required=True, dest='files',
+                        help="File containing the list of modified files, one per line")
+
+    parser.add_argument("-e", "--event", type=str, required=True, dest='event',
+                        choices=['added', 'modified', 'deleted'],
+                        help="Type of alert's event that we expect: added, modified, deleted")
+
+    parser.add_argument("-l", "--log_json", type=str, required=False, dest='log_json_path',
+                        help="alerts.json path. default value '/var/ossec/logs/alerts/alerts.json'",
+                        default="/var/ossec/logs/alerts/alerts.json")
+
+    parser.add_argument("-o", "--output_path", type=str, required=False, dest='output_path',
+                        help="Output path for missing files alerts.",
+                        default="debug_missing_file_alerts.log")
+    args = parser.parse_args()
+    paths_list_set = paths_acquisition(args.files)
+    pruned_alerts_set = alerts_prune(args.log_json_path, args.event)
+    sub_paths = paths_list_set - pruned_alerts_set
```

Comment:
What happens if there is other alerts besides the ones that we generated? 
Will the test fail even if al the alerts indicated in `files` are present?

Reply:
Hello @DFolchA,

Alerts and filnames are both sets. The difference of sets solves this use case. 

Thanks for your review !